pyproject.toml

```diff
@@ -23,7 +23,7 @@ packages = [{ include = "django_twined" },]
 python = ">=3.9,<3.11"
 django = ">=3.0,<4.0"
 channels = ">=3.0,<4.0"
-octue = ">=0.40.0,<1"
+octue = {git = "https://github.com/octue/octue-sdk-python.git", rev = "publish-answers-to-question-topic-2"}
```

Comment:
We'll need to bump this on release

Reply:
The commit to update this needs to be labelled with a BREAKING-CHANGE so that semver works.

Reply:
> We'll need to bump this on release

Yep for sure

> The commit to update this needs to be labelled with a BREAKING-CHANGE so that semver works.

Will do